src/test/codegen/c-variadic.rs

```diff
@@ -52,7 +52,6 @@ pub unsafe extern "C" fn c_variadic(n: i32, mut ap: ...) -> i32 {
         sum += ap.arg::<i32>();
     }
     sum
-    // CHECK: call void @llvm.va_end
```

Comment:
Why was this removed? Could you add this back?

Reply:
This test runs with `-C no-prepopulate-passes`, so the `drop` destructor doesn't get inlined, and `llvm.va_end` doesn't show here. In normal code, it would show up.

I can add it to the other tests.

Reply:
Could you write a `CHECK` for the `Drop` function instead then. E.g. would something like what is done in [src/test/codegen/drop.rs](https://github.com/rust-lang/rust/blob/master/src/test/codegen/drop.rs) work?